example/fluentd-multi.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentd
+  namespace: dynatrace
+  labels:
+    k8s-app: fluentd-logging
+    version: v1
+spec:
+  selector:
+    matchLabels:
+      k8s-app: fluentd-logging
+      version: v1
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentd-logging
+        version: v1
+    spec:
+      serviceAccountName: dynatrace-monitoring
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      initContainers:
+      - name: fluentd-permission-fix
+        image: busybox
+        command: ["sh","-c","touch /var/log/fluentd.pos; chmod -R 770 /var/log/fluentd.pos; "]
+        volumeMounts:
+        - name: var-log
+          mountPath: /var/log/
+      containers:
+      - name: fluentd
+        image: mreider/fluentd:2
```

Comment:
They should really build their own image, I think
`${YOUR_FLUENTD_IMAGE}`

Reply:
that's fine.